gdx/src/com/badlogic/gdx/scenes/scene2d/ui/SpinnerDecimalModel.java

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.scenes.scene2d.ui;
+
+import java.math.BigDecimal;
+
+import com.badlogic.gdx.scenes.scene2d.ui.Spinner.SpinnerModel;
+
+/** Spinner model used to display decimal numbers
+ * @author Jeremy Gillespie-Cloutier */
+public class SpinnerDecimalModel extends SpinnerModel<BigDecimal> {
```

Comment:
Games probably won't use BigDecimals much, floats (or doubles, if you insist) should be enough.

By the way, `toString` is much better than ` + ""` (for example `Integer.toString(int)` for primitives), since `toString` allocates only String, but `+ ""` has to allocate `StringBuilder` and then `String`.